packages/jest-runtime/src/index.ts

```diff
@@ -2123,6 +2123,7 @@ export default class Runtime {
           );
         }
       },
+      getSeed: () => this._environment.seed,
```

Comment:
Throw an error if we `this._environment.seed` is `undefined` (which will happen if people have mismatched versions of Jest's deps)